lib/private/Files/Filesystem.php

```diff
@@ -1089,4 +1089,19 @@ public static function getOwner($path) {
 	public static function getETag($path) {
 		return self::$defaultInstance->getETag($path);
 	}
+
+	/**
+	 * Determines weather primary object storage is enabled on the instance
```

Comment:
```suggestion
	 * Determines whether primary object storage is enabled on the instance
```

"the weather is cold", "whether or not to do it" are different words ;)

But I don't know if it is worth running a whole run of CI just for a 2-char change in a comment!

Reply:
commit it, it is worth. we should not look as fools…